grafana/resource_alert_notification.go

```diff
@@ -57,6 +57,11 @@ func ResourceAlertNotification() *schema.Resource {
 				Optional:  true,
 				Sensitive: true,
```

Comment:
With the addtion of `secure_settings`, could `settings` be `Sensitive: false`?

Reply:
People might still decide to put the apikey in settings

Reply:
It would be nice to see the diffs in `settings` in the plan, but I see your point.

Reply:
People *might* still decide to commit API keys as well. 😋 

People *should* not place API keys in `settings` once `secure_settings` is available, right? 
If `settings.Sensitive` remains `true`, it's quite hard to see any non-sensitive changes, such as adding or removing e-mail addresses.

If you use Terraform 0.14 with the enabled experiment, or you've upgraded to 0.15 and [sensitive variables are hidden by default](https://www.terraform.io/docs/extend/best-practices/sensitive-state.html#using-the-sensitive-flag), you *could* reasonably expect that your secret values remain hidden in the CLI output. Fair enough. 

**Alternative**: bump the major version.

You could consider the possibility of exposing sensitive values in the CLI a breaking change, even though 0.15 hasn't been out for long, and allow those who expect non-encrypted, non-sensitive and sensitive values alike to remain hidden to continue to do so. 
If they choose to upgrade, they would (hopefully) read the release notes informing them that they must move their sensitive values to `secure_settings` if they wish for them to remain hidden.
That way, those who want to see their non-sensitive changes can do so, and we can have our cake and eat it too.

Then again, people *might* still decide to do a major version upgrade without reading the release notes.

Reply:
@trotttrotttrott  any preferences on the above? I can add it to the PR if you want.